### PR TITLE
Added 7 markdown files for beginner documentation of Apache Solr

### PR DESCRIPTION
**Pull request by shannee-07**

### CODE REVIEW

**Comment by AyanF** [comment]
Appropriate filename for Solr-Terminologies.md would be Glossary.md, It should in the parent directory (search/solr)

**Comment by shannee-07** [comment]
> Appropriate filename for Solr-Terminologies.md would be Glossary.md, It should in the parent directory (search/solr)

Thank you for your suggestions! I've implemented the changes.

**Comment by shannee-07** [comment]
Marked the Glossary terms with superscript

File: search/solr/Glossary.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+### 7. Config Set:
+
+A configuration set in Solr defines the schema structure, indexing settings, and other configurations used by cores within a collection. It includes details such as field definitions, analysis chains, update request handlers, and more. Config sets provide a way to manage and share common configurations across multiple cores or collections within a SolrCloud cluster.
+
+### 8. ZooKeeper: 
+
+ZooKeeper is a centralized service for maintaining configuration information, providing distributed synchronization, and managing cluster state for distributed systems like SolrCloud. In a SolrCloud deployment, ZooKeeper manages cluster coordination tasks such as leader election, shard allocation, and tracking the status of individual Solr nodes. It stores metadata about collections, shards, and replicas, enabling Solr nodes to dynamically join or leave the cluster without affecting its availability or data integrity.
+
+
+
```
Comment (AyanF) [inline search/solr/Glossary.md:38]: Please move this content to Solr Cloud

File: search/solr/Glossary.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## Solr terminologies and common confusions:
```
Comment (AyanF) [inline search/solr/Glossary.md:1]: Update Header to Glossary

Comment (shannee-07) [reply search/solr/Glossary.md]: > Update Header to Glossary

Updated!

**Pull request merged**